src/EA.Weee.Integration.Tests/Handlers/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -86,7 +86,7 @@ public class WhenIGetAnAatfsSummaryData : GetAatfSummaryRequestHandlerTestBase
                     n.UpdateStatus(NoteStatus.Rejected, UserId.ToString());
                 }).Create();
 
-                request = new GetAatfSummaryRequest(aatf.Id);
+                request = new GetAatfSummaryRequest(aatf.Id, 2022);
```

Comment:
Similar to the comment above with the data access integration test. Update the integration test EvidenceNoteDbSetup to have a WithComplianceYear method that updates the compliance year for a note. Then create approved notes outside of the compliance year to ensure only expected notes are include in the total.